src/psmove_orientation.c

```diff
@@ -27,7 +27,9 @@
  * POSSIBILITY OF SUCH DAMAGE.
  **/
 
-
+#ifdef _MSC_VER
+#define __func__ __FUNCTION__
```

Comment:
[This answer](http://stackoverflow.com/questions/4384765/whats-the-difference-between-pretty-function-function-func) on SO suggests that it should be MSVC-specific because __func__ is preferred but not available in MSVC.